minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs

```diff
@@ -63,11 +63,26 @@ protected override bool Execute(string name, CefV8Value obj, CefV8Value[] argume
 
                     var context = CefV8Context.GetCurrentContext();
 
-                    var v8Callback = arguments != null && arguments.Length != 0 && arguments[arguments.Length - 1].IsFunction
-                        ? arguments[arguments.Length - 1]
-                        : null;
-                    var parameters = v8Callback != null ? arguments.Take(arguments.Length - 1).ToArray() : arguments;
+                    var v8Callback = CefV8Value.CreateUndefined();
+                    var parameters = arguments;
 
+                    if (arguments != null && arguments.Length != 0 && arguments[arguments.Length - 1].IsFunction)
+                    {
+                        v8Callback = arguments[arguments.Length - 1];
+                        parameters = v8Callback != null ? arguments.Take(arguments.Length - 1).ToArray() : arguments;
+                    }
+                    else
+                    {
+                        if (arguments != null && arguments.Length != 0 && arguments[0].IsFunction)
+                        {
+                            v8Callback = arguments[0];
+                            parameters = v8Callback != null && arguments.Length > 1 ? arguments.Skip(1).ToArray() : arguments;
+                        }
+                        else
+                        {
```

Comment:
Yes, this is needed. This handler is used on all JS method calling. If we remove it, we may have problem with other methods.